ocamltest/OCAMLTEST.org

```diff
@@ -711,6 +711,10 @@ output:
  ... testing 'hello.ml' with 1 (script) => failed (Why should this pass in the first place)
 #+end_src
 
+In addition, the script can write lines of the form =variable="string"=, or
+=variable+="string"=, or =-variable=, to file =${ocamltest_response}=, to set, append to,
+and unset ocamltest environment variables, respectively.
```

Comment:
`=variable="string"=` does not render correctly on github. Is there some org-mode escaping syntax for code that contains equal signs?

Reply:
From the PR description I’m not sure to see: are there intrinsic advantages to AsciiDoc over `.org` and Markdown? To me the advantage of AsciiDoc is that it looks closer to Markdown and thus slightly less obscure (to me) than org-mode. But then, why not convert this document to Markdown instead?

Reply:
Independently of that, I replace the equal signs with tildas and it works.